### ISSUE TITLE
Function tweaks

### ISSUE THREAD

**Issue by mdozmorov** [stilianoudakis/preciseTAD#7]
Should we set the default for "signal" argument in `bedToGRangesList`? It is typically column 5, not 4 like in the example.

`TADrfe` - "Backwards elimination is performed from 2 to p" - should it be "from p to 2"? 

`TADrfe` - Describe columns of the output, in "Value" section of the function's help

When setting `juicer = TRUE`, the output does not change. Describe more in the function's help and the vignette.

**Comment by stilianoudakis** [comment]
Should we set the default for "signal" argument in bedToGRangesList? It is typically column 5, not 4 like in the example.

Added the default for signal to 5. However, I kept the example the same

TADrfe - "Backwards elimination is performed from 2 to p" - should it be "from p to 2"?

Correct. Edited the roxygen skeleton to match

TADrfe - Describe columns of the output, in "Value" section of the function's help

Added

When setting juicer = TRUE, the output does not change. Describe more in the function's help and the vignette.

When you set juicer=TRUE the 2nd element in the list of results from preciseTAD should be a dataframe (instead of a GRanges object). Are you saying that this is not what you get? Or are you saying to just add more clarification? If the latter, I added clarification in the help file. Let me know if it is the former because I don't have that problem on my end
